core/templates/dev/head/pages/base.html

```diff
@@ -134,68 +134,69 @@
-                  </h2>
-                  <p translate="I18N_SPLASH_JAVASCRIPT_ERROR_DESCRIPTION"
-                     translate-values="{hrefUrl: 'http://www.enable-javascript.com/'}">Oppia is a free, open-source learning platform full of interactive activities called 'explorations'.  Sadly, Oppia requires JavaScript to be enabled in your web browser in order to function properly and your web browser has JavaScript disabled.  If you need help enabling JavaScript, <a href="http://www.enable-javascript.com"/>click here.</a></p>
-                  <p translate="I18N_SPLASH_JAVASCRIPT_ERROR_THANKS">Thank you.</p>
-                </div>
+            {{ warnings_and_loader() }}
+          </div>
+
+          <noscript>
+            <div class="oppia-page-cards-container">
+              <div class="md-default-theme oppia-page-card oppia-long-text">
+                <!-- Note to developers: We replicate the translated text inline because, without JavaScript enabled, the translation engine doesn't kick in.-->
+                <h2>
+                  <span translate="I18N_SPLASH_JAVASCRIPT_ERROR_TITLE">We Need JavaScript in Your Browser</span>
+                  <i class="material-icons">&#xE811;</i>
+                </h2>
+                <p translate="I18N_SPLASH_JAVASCRIPT_ERROR_DESCRIPTION"
+                translate-values="{hrefUrl: 'http://www.enable-javascript.com/'}">Oppia is a free, open-source learning platform full of interactive activities called 'explorations'.  Sadly, Oppia requires JavaScript to be enabled in your web browser in order to function properly and your web browser has JavaScript disabled.  If you need help enabling JavaScript, <a href="http://www.enable-javascript.com"/>click here.</a></p>
+                <p translate="I18N_SPLASH_JAVASCRIPT_ERROR_THANKS">Thank you.</p>
               </div>
-            </noscript>
+            </div>
+          </noscript>
 
-            <side-navigation-bar></side-navigation-bar>
-          </div>
+          <side-navigation-bar></side-navigation-bar>
         </div>
       </div>
+    </div>
 
-      {% if DEV_MODE %}
-        <div class="oppia-dev-mode">
-          Dev Mode
-        </div>
-      {% endif %}
+    {% if DEV_MODE %}
+    <div class="oppia-dev-mode">
+      Dev Mode
+    </div>
+    {% endif %}
 
-      {% if SITE_FEEDBACK_FORM_URL %}
-        <a href="{{SITE_FEEDBACK_FORM_URL}}" target="_blank"
-           class="oppia-site-feedback oppia-transition-200">
-          <i class="material-icons md-18">&#xE87F;</i>
-            <span translate="I18N_SPLASH_SITE_FEEDBACK"></span>
-          </i>
-        </a>
-      {% endif %}
+    {% if SITE_FEEDBACK_FORM_URL %}
+    <a href="{{SITE_FEEDBACK_FORM_URL}}" target="_blank"
+      class="oppia-site-feedback oppia-transition-200">
+      <i class="material-icons md-18">&#xE87F;</i>
+      <span translate="I18N_SPLASH_SITE_FEEDBACK"></span>
+    </i>
+    </a>
+    {% endif %}
```

Comment:
Again something seems to be wrong with the indentation. The existing stuff seemed good; you might want to leave that as it was.

(Except for the closing i tag on line 197, which actually seems completely superfluous.)